tests/test-utils.tsx

```diff
@@ -4,9 +4,14 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import { RouterContext } from 'next/dist/shared/lib/router-context';
 import { NextRouter } from 'next/router';
-import { ThemeProvider } from '@material-ui/core';
+import { ThemeProvider, Theme, StyledEngineProvider } from '@mui/material';
 import theme from '../lib/theme';
 
+declare module '@mui/styles/defaultTheme' {
```

Comment:
Also similar, we should only need to declare this once right? We have this in the `pages/_app.tsx` file.